Taarafo.Portal.Web.Tests.Unit/Services/Foundations/Groups/GroupServiceTests.Exceptions.RetrieveById.cs

```diff
@@ -0,0 +1,143 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Moq;
+using RESTFulSense.Exceptions;
+using Taarafo.Portal.Web.Models.Groups;
+using Taarafo.Portal.Web.Models.Groups.Exceptions;
+using Xunit;
+
+namespace Taarafo.Portal.Web.Tests.Unit.Services.Foundations.Groups
+{
+    public partial class GroupServiceTests
+    {
+        [Theory]
+        [MemberData(nameof(CriticalDependencyExceptions))]
+
```

Comment:
remove extra line

Reply:
I removed the extra line in the Code Rub.